examples/llm_zero_shot_learning/simple_model_training.py

```diff
@@ -50,40 +50,40 @@
 
 config = yaml.safe_load(
     """
-        input_features:
-            - name: review
-              type: text
-        output_features:
-            - name: label
-              type: category
-              preprocessing:
-                vocab: [positive, neutral, negative]
-                fallback_label: neutral
-                prompt_template: |
-                    Context information is below.
-                    ###
-                    {review}
-                    ###
-                    Given the context information and not prior knowledge, classify the context as one of: {labels}
-              decoder:
-                match:
-                    positive:
-                        type: contains
-                        value: positive
-                    neutral:
-                        type: regex
-                        value: neutral
-                    negative:
-                        type: contains
-                        value: negative
-        model_type: llm
-        generation_config:
-            temperature: 0.1
-            top_p: 0.75
-            top_k: 40
-            num_beams: 4
-            max_new_tokens: 5
-        model_name: facebook/opt-350m
+model_type: llm
+model_name: facebook/opt-350m
+generation:
+    temperature: 0.1
+    top_p: 0.75
+    top_k: 40
+    num_beams: 4
+    max_new_tokens: 64
+input_features:
+-
+    name: review
+    type: text
+    preprocessing:
+        lowercase: false
+        prompt:
+            task: "Classify the sample input as either negative, neutral, or positive."
+output_features:
+-
+    name: label
+    type: category
+    preprocessing:
+        fallback_label: "neutral"
```

Comment:
Did we move this into the decoder?

Reply:
Not yet– this felt out of scope for this PR (which is already big enough as-is). We can fast-follow with that change.